Gemfile

```diff
@@ -2,4 +2,8 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+group :test, :development do
+  gem 'pry-byebug'
```

Comment:
This does not belong here. If you want to add dependencies like this to a Gem, even if for development purposes, do it in the gemspec.

Also, byebug is already a development dependency. Why this extra dependency? If you want to step through things, why not just use byebug in the first place?